specification/2.0/schema/image.schema.json

```diff
@@ -29,5 +28,8 @@
     "additionalProperties": false,
     "dependencies": {
         "bufferView": [ "mimeType" ]
+    },
+    "not": {
```

Comment:
I think anyOf here is clearer than using not? See http://stackoverflow.com/questions/31839578/how-to-define-a-json-schema-that-requires-at-least-one-of-many-properties